docs/data/base/getting-started/usage/usage.md

```diff
@@ -1,4 +1,4 @@
-# MUI Base - Usage
+# Usage
```

Comment:
The title would be "MUI Base - Usage | MUI Base". The duplication feels noisy in this context.